code/go/0chain.net/blobbercore/allocation/entity.go

```diff
@@ -302,18 +290,3 @@ type ReadPoolRedeem struct {
 	PoolID  string `json:"pool_id"` // read pool ID
 	Balance int64  `json:"balance"` // balance reduction
 }
-
-// SubReadRedeemed subtracts tokens redeemed from read pools.
-func SubReadRedeemed(rps []*ReadPool, redeems []ReadPoolRedeem) {
```

Comment:
why do you remove redeem logic?

Reply:
it was a tricky one, I just remembered. Basically we didn't have bunch of pool ids anymore, so the information here is no goo to us. So my theory is we could probably update read pool by aggregating the the cost here and then subtracting from appropriate account in read-pool table. But I didn't end up doing it as things seems to have been working as is.
We still update the existing read pool with new values from the chain in the method `(rm *ReadMarkerEntity) Sync(ctx context.Context)`